cdhweb/urls.py

```diff
@@ -48,7 +48,7 @@
     path("_500/", lambda _: 1 / 0),  # for testing 500 error page
     # main apps
     path("people/", include("cdhweb.people.urls", namespace="people")),
-    path("blog/", include("cdhweb.blog.urls", namespace="blog")),
+    path("", include("cdhweb.blog.urls", namespace="blog")),
```

Comment:
Is this still mounting them under `blog`?

Reply:
no the url looks like https://cdh.dev.springload.nz/blog-landing-page/2024/06 etc
I was thinking it is better to use the slug because we will have more than one blog landing page

Reply:
Oh yep, missed that we were using the slug there. Good call